src/main/java/edu/metrostate/ics499/team2/controllers/QuizController.java

```diff
@@ -48,8 +48,8 @@ public Quiz create(@RequestBody final Quiz quiz) {
 
 	}
 
-	@GetMapping("/getByUserId")
-	public List<Quiz> getByUserId(@RequestParam String userId) { return this.quizService.findQuizByUserId(userId); }
+	@GetMapping("/getbyuserid/{userId}")
+	public List<Quiz> getByUserId(@PathVariable("userId") String userId) { return this.quizService.findQuizByUserId(userId); }
 
```

Comment:
oh yah, thanks I forgot I made this update to get the front-end quiz system to work